app/api/__init__.py

```diff
@@ -162,7 +159,7 @@
 
 # tax
 api.route(TaxList, 'tax_list', '/events/<int:id>/tax', '/events/<identifier>/tax')
-api.route(TaxDetail, 'tax_detail', '/tax/<int:id>', '/events/<int:event_id>/tax')
+api.route(TaxDetail, 'tax_detail', '/tax/<int:id>')
```

Comment:
@enigmaeth the first link(TaxList) is supposed to be only for POST. Otherwise GET should be done from TaxDetail only.